tests/lib/file-finder.js

```diff
@@ -88,6 +88,22 @@ describe("FileFinder", function() {
                 assert.equal(actual, expected);
             });
         });
+
+        describe("Not consider directory with expected file names", function() {
+
+            it("should not be found, and an empty array returned", function() {
+                var expected = path.join(process.cwd(), "package.json");
+                process.chdir(fileFinderDir);
+                var finder = new FileFinder("package.json");
```

Comment:
I have to, to hit a particular scenario. 
I dont see any problems. I can put the revert inside the `afterEach` so that it would always put the path back to original but other than that I dont see any issues.